crates/host/src/wasmbus/mod.rs

```diff
@@ -3106,58 +3120,61 @@ impl Host {
             "stop provider"
         );
 
+        let annotations: Annotations = annotations.unwrap_or_default().into_iter().collect();
         let mut providers = self.providers.write().await;
         let hash_map::Entry::Occupied(mut entry) = providers.entry(provider_ref.clone()) else {
             return Ok(SUCCESS.into());
         };
         let provider = entry.get_mut();
         let instances = &mut provider.instances;
         if let hash_map::Entry::Occupied(entry) = instances.entry(link_name.clone()) {
```

Comment:
Should we add an `else` to this that emits a stop-failed event because nothing matched? (same for scaling down/stopping actors)

Reply:
I don't even know if we have a stop_failed event! That would be a decent use for the new event and it would be nice to know when you did something wrong, though I would lean more in the direction of returning a failed `CtlOperationAck`

Reply:
Agreed, I think a failed ack is a good addition